src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -967,6 +981,20 @@ static grpc_lb_policy *glb_create(grpc_exec_ctx *exec_ctx,
   glb_policy->args = grpc_channel_args_copy_and_add_and_remove(
       args->args, args_to_remove, GPR_ARRAY_SIZE(args_to_remove), &new_arg, 1);
 
+  /* Fallback is off */
+  glb_policy->fallback = false;
+
+  /* Copy the backend server addresses from the resolver for fallback */
+  grpc_lb_addresses *backends = grpc_lb_addresses_create(
+      addresses->num_addresses - num_grpclb_addrs, NULL);
+  for (size_t addr_copied = 0, i = 0; i < addresses->num_addresses; ++i) {
+    if (!addresses->addresses[i].is_balancer) {
+      memcpy(backends->addresses + addr_copied++, addresses->addresses + i, 
```

Comment:
I think we are talking about the same functionality, but using different names.

If populate means fill, I think we should avoid using `..._copy()`? Because the existing `grpc_lb_addresses_copy()` returns a new object, which looks reasonable. (So does `user_data_vtable->copy()`.) 

`..._set()` may be a confusing name. Ah, naming is so difficult...